frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -1,24 +1,31 @@
 import React, { ReactElement } from 'react';
-import ApartmentImg from '../../assets/apartment-sample.png';
+import ApartmentImg from '../../assets/apartment-placeholder.png';
 import { Card, CardContent, CardMedia, Grid, makeStyles, Typography } from '@material-ui/core';
 import { Apartment } from '../../../../common/types/db-types';
+import HeartRating from '../utils/HeartRating';
 
 type Props = {
   buildingData: Apartment;
   numReviews: number;
   company?: string;
 };
-
 const useStyles = makeStyles({
   img: {
     height: 200,
-    width: '100%',
+    width: 200,
```

Comment:
Try not to use absolute sizing — if you resize the page right now, you'll find that the words spill over onto the image. If you change this to percentages, it should preserve responsive sizing!